src/components/EditProfileModal.tsx

```
@@ -4,11 +4,24 @@ import { Input } from './ui/input';
 import { Textarea } from './ui/textarea';
 import { Label } from './ui/label';
 import { ImageWithFallback } from './figma/ImageWithFallback';
-import { X, Upload, Trash2, Lock } from 'lucide-react';
+import {
+  X,
+  Upload,
+  Trash2,
+  Lock,
+  Map,
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)